### PR TITLE
HomeTV App: added TV remote control

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1277
[Problem] Add TV remote widget
[Solution] TV remote widget is not implemented

[Demo screen]
image
image

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: examples/mobile/HomeApp/css/style.css [hunk 1]
```diff
@@ -34,15 +34,80 @@
...
+    flex-direction: column;
+    border-radius: 20px;
+    box-shadow: 1px 1px 3px 0 var(--overlay);
+    margin: 5px;
+    font-size: 16px;
+    background-image: linear-gradient(170deg,
+        rgba(0, 0, 0, 0.2) 0%, var(--background-area-color) 15%,
+        var(--background-area-color) 85%, rgba(0, 0, 0, 0.2) 100%);
+}
+.app-chanel-control {
```
Comment (korneliakobiela) [inline examples/mobile/HomeApp/css/style.css]: TYPO 
``` css
.app-channel-control
```

File: examples/mobile/HomeApp/css/style.css [hunk 2]
```diff
@@ -34,15 +34,80 @@
...
     margin-left: 20px;
-    margin-right: 12px;    
+    margin-right: 12px;
 }
 .app-btn-icon-burger::after {
     -webkit-mask-image: url(../images/Hamburger_icon.svg);
     mask-image: url(../images/Hamburger_icon.svg);
 }
 
+
```
Comment (korneliakobiela) [inline examples/mobile/HomeApp/css/style.css]: Please, remove unnecessary blank lines

File: examples/mobile/HomeApp/css/style.css [hunk 3]
```diff
@@ -34,15 +34,80 @@
...
+    flex-direction: column;
+    border-radius: 20px;
+    box-shadow: 1px 1px 3px 0 var(--overlay);
+    margin: 5px;
+    font-size: 16px;
+    background-image: linear-gradient(170deg,
+        rgba(0, 0, 0, 0.2) 0%, var(--background-area-color) 15%,
+        var(--background-area-color) 85%, rgba(0, 0, 0, 0.2) 100%);
+}
+.app-four-way-direction {
```
Comment (DohyungLim) [inline examples/mobile/HomeApp/css/style.css]: four-way => 4way.

I think that it looks good.

File: examples/mobile/HomeApp/index.html [hunk 4]
```diff
@@ -43,6 +43,32 @@
...
+						<a href="#" class="ui-btn ui-btn-icon app-btn-icon-volume-down"  data-icon="minus" data-style="flat"></a>		
+					</div>
+					<div class="app-four-way-direction">
+						<a href="#" class="ui-btn ui-btn-icon app-btn-4way-up" data-icon="up" data-style="flat"></a>
+						<a href="#" class="ui-btn ui-btn-icon app-btn-4way-right" data-icon="right" data-style="flat"></a>
+						<a href="#" class="ui-btn app-btn-4way-ok" data-style="flat">OK</a>
+						<a href="#" class="ui-btn ui-btn-icon app-btn-4way-left" data-icon="left" data-style="flat"></a>
+						<a href="#" class="ui-btn ui-btn-icon app-btn-4way-down" data-icon="down" data-style="flat"></a>
+					</div>
+					<div class="app-chanel-control">
```
Comment (DohyungLim) [inline examples/mobile/HomeApp/index.html]: channel is correct spelling.

**Review by DohyungLim — COMMENTED** [review]
2 comment.

File: examples/mobile/HomeApp/index.html [hunk 5]
```diff
@@ -43,6 +43,32 @@
 					!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src='https://weatherwidget.io/js/widget.min.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','weatherwidget-io-js');
 				</script>
 			</div>
+
+			<div class="ui-content-subheader">
+				TV remote Control
```
Comment (lslachciak) [inline examples/mobile/HomeApp/index.html:48]: We discussed with @singa2000 that this widget should replace Weather widget.

**Review by lslachciak — COMMENTED** [review]
Just please follow reviewer comments. Widget looks good

File: examples/mobile/HomeApp/index.html (hunk 5 above)
Comment (singa2000) [reply examples/mobile/HomeApp/index.html:48]: no no~ it's not need in this sample.
If I needed, I'll replace it.
Keeping the weather widget and keeping the TV remote contol widet, please?

Comment (lslachciak) [reply examples/mobile/HomeApp/index.html:48]: ok, sorry for misunderstanding

File: examples/mobile/HomeApp/css/style.css (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/css/style.css]: Done.

File: examples/mobile/HomeApp/css/style.css (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/css/style.css]: Done

File: examples/mobile/HomeApp/css/style.css (hunk 3 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/css/style.css]: Done

File: examples/mobile/HomeApp/index.html (hunk 4 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/index.html]: Done.

**Review by singa2000 — APPROVED** [review]
It looks good.
Thanks

**Pull request merged**